storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operations/execute/PreparedExecuteSQL.java

```diff
@@ -113,7 +118,22 @@ public Object executeAsBlocking() {
     @CheckResult
     @Override
     public Observable<Object> asRxObservable() {
-        return RxJavaUtils.createObservable(storIOSQLite, this);
+        throwExceptionIfRxJavaIsNotAvailable("asRxObservable()");
+
+        final Set<String> observesTables = rawQuery.observesTables();
+        final Set<?> observesTags = rawQuery.observesTags();
+
+        final Observable<Object> observable;
+        if (!observesTables.isEmpty() || !observesTags.isEmpty()) {
+            observable = ChangesFilter.applyForTablesAndTags(storIOSQLite.observeChanges(), observesTables, observesTags)
```

Comment:
Please correct me if I'm wrong, but it was our design decision, since `executeSql` is basically a side effect and does not return data from database, same as we do for Put and Delete, we don't subscribe them to db changes!